test/test_multidevice_sharding.cpp

```diff
@@ -18,6 +18,46 @@
 
 namespace nvfuser {
 
+// Test single device computation with device parallelized axis is handled
```

Comment:
Why not testing also on non-trivial meshes ?

Reply:
Good point, will update to a non-trivial mesh.